src/ttauri/console_win32.cpp

```diff
@@ -48,7 +49,7 @@ void console_output(std::string_view text, std::ostream &output) noexcept
     tt_assert(std::addressof(output) == std::addressof(std::cout) || std::addressof(output) == std::addressof(std::cerr));
 
     if (IsDebuggerPresent()) {
-        ttlet text_ = to_wstring(text);
+        std::wstring text_(text.begin(), text.end());
```

Comment:
This does not do proper UTF-8 to wstring conversion. It simply copies the code-unit values.

The to_wstring() that does proper conversion is in the "codec/UTF.hpp" which is included in "strings.hpp" (or string.hpp I forget).

Also I dislike that form of value initialisation; it looks to much like a function call to me. I would accept:
```
ttlet text_ = std::wstring{text.begin(), text.end()};
ttlet text_ = std::wstring(text.begin(), text.end());
auto text_ = std::wstring{text.begin(), text.end()};
auto text_ = std::wstring(text.begin(), text.end());
std::wstring text_ = std::wstring{text.begin(), text.end()};
std::wstring text_ = std::wstring(text.begin(), text.end());
std::wstring text_ = {text.begin(), text.end()};
```

I guess I always like to see an assignment operator when initialising a variable. I guess I should add that to the code style documentation.